dist/s_clang_scan

```diff
@@ -6,7 +6,9 @@ setup_trap 'rm -rf $t_out $t_out_filtered $t_build $t_diff_filtered'
 
 t_out=__wt.$$.out
 t_out_filtered=__wt.$$.out_filtered
+t_out_errors=__wt.$$.out_errors
 t_diff_filtered=__wt.$$.diff_filtered
+t_diff_result=__wt.$$.diff_result
```

Comment:
Can you also add the new intermediate files `t_out_errors` and `t_diff_result` to `setup_trap` for cleanup?

Or maybe just replace the list to `__wt.$$.*`?

Reply:
Done!